ni_measurementlink_service/_internal/service_manager.py

```diff
@@ -53,7 +53,7 @@ def start(
         configuration_parameter_list: List[ParameterMetadata],
         output_parameter_list: List[ParameterMetadata],
         measure_function: Callable,
-    ) -> str:
+    ) -> Optional[str]:
```

Comment:
This is a step in the wrong direction. If start() fails, it should throw an exception.

Reply:
I have changed back to `str` because instead of returning `None`, I've updated to `raise` the caught exception from called methods.